configure.ac

```diff
@@ -1226,9 +1226,10 @@ AS_CASE([$host/$ac_cv_cc_name],
   [s390x-*-linux-gnu/gcc],           [PY_SUPPORT_TIER=3], dnl Linux on 64bit s390x (big endian), glibc, gcc
   [x86_64-*-freebsd*/clang],         [PY_SUPPORT_TIER=3], dnl FreeBSD on AMD64
   [aarch64-apple-ios*-simulator/clang],   [PY_SUPPORT_TIER=3], dnl iOS Simulator on arm64
-  [aarch64-apple-ios*/clang],             [PY_SUPPORT_TIER=3], dnl iOS on ARM64
+  [aarch64-apple-ios*/clang],        [PY_SUPPORT_TIER=3], dnl iOS on ARM64
```

Comment:
I think it would be better to indent the other entries more to line up with the line containing `aarch64-apple-ios*-simulator/clang`